public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -82,17 +82,7 @@ export function DashList(props: PanelProps<DashListOptions>) {
     fetchDashboards(props.options, props.replaceVariables).then((dashes) => {
       setDashboards(dashes);
     });
-  }, [
-    props.options.showSearch,
-    props.options.showStarred,
-    props.options.showRecentlyViewed,
-    props.options.maxItems,
-    props.options.query,
-    props.options.tags,
-    props.options.folderId,
-    props.replaceVariables,
-    props.renderCounter,
```

Comment:
Nit: I think we might want to keep the `renderCounter` since that is used on the angular side to trigger a re-render. But with that said I have not tested this code so I'm not 100% sure.